soc/realtek/ec/rts5912/debug_swj.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2024 Realtek Semiconductor Corporation, SIBG-SD7
+ * Author: Titan Chen <titan.chen@realtek.com>
+ */
+
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/init.h>
+
+#define SWJ_NODE DT_NODELABEL(swj_port)
+
+PINCTRL_DT_DEFINE(SWJ_NODE);
+
+const struct pinctrl_dev_config *swj_pcfg = PINCTRL_DT_DEV_CONFIG_GET(SWJ_NODE);
+
+static int swj_connector(void)
+{
+	int err;
+
+	err = pinctrl_apply_state(swj_pcfg, PINCTRL_STATE_DEFAULT);
```

Comment:
This requires the pinctrl support you add with a later commit.  [driver: pinctrl: Add pin control initial version of RTS5912](https://github.com/zephyrproject-rtos/zephyr/pull/75267/commits/7da12fca0b2f11e5c5346963f06ee43698ae7fad).

You should add the `swj_port` driver support to a commit after the pinctrl commit.

Reply:
Fix